DFC.ServiceTaxonomy.GraphSync/Orchestrators/DeleteOrchestrator.cs

```diff
@@ -77,15 +77,26 @@ public async Task<bool> Delete(ContentItem contentItem)
 
             var deleteGraphSyncers = await Task.WhenAll(
                 GetDeleteGraphSyncerIfDeleteAllowed(
-                    contentItem, _publishedContentItemVersion, DeleteOperation.Delete),
+                    contentItem, _publishedContentItemVersion, SyncOperation.Delete),
                 GetDeleteGraphSyncerIfDeleteAllowed(
-                    contentItem, _previewContentItemVersion, DeleteOperation.Delete));
+                    contentItem, _previewContentItemVersion, SyncOperation.Delete));
 
-            (SyncStatus publishedSyncStatus, IDeleteGraphSyncer? publishedDeleteGraphSyncer) = deleteGraphSyncers[0];
-            (SyncStatus previewSyncStatus, IDeleteGraphSyncer? previewDeleteGraphSyncer) = deleteGraphSyncers[1];
+            (IAllowSyncResult publishedAllowSyncResult, IDeleteGraphSyncer? publishedDeleteGraphSyncer) = deleteGraphSyncers[0];
+            (IAllowSyncResult previewAllowSyncResult, IDeleteGraphSyncer? previewDeleteGraphSyncer) = deleteGraphSyncers[1];
 
-            if (publishedSyncStatus == SyncStatus.Blocked || previewSyncStatus == SyncStatus.Blocked)
+            if (publishedAllowSyncResult.AllowSync == SyncStatus.Blocked || previewAllowSyncResult.AllowSync == SyncStatus.Blocked)
             {
+                var graphBlockers = new List<(string GraphReplicaSetName, IAllowSyncResult AllowSyncResult)>();
+                if (publishedAllowSyncResult.AllowSync == SyncStatus.Blocked)
```

Comment:
Just an observation and I know it's existing code, but publishedAllowSyncResult.AllowSync == SyncStatus.Blocked doesn't read quite right to me. Might be better if we had something like publishedAllowSyncResult.Status or publishedAllowSyncResult.SyncStatus. In my head AllowSync would indicate a bool

Reply:
I'll change it in the next PR to ..`AllowSync.Result == AllowSyncResult.` as discussed.